src/selectors/authenticate.js

```diff
@@ -2,4 +2,4 @@ export const hasUser = (state) => !!state.authenticate.user
 
 export const selectUser = (state) => state.authenticate.user
 
-export const selectGroups = (state) => state.authenticate.user.groups
+export const selectGroups = (state) => state.authenticate.user?.groups
```

Comment:
this allows our new component to work since it both uses the group selector and it runs on the home page, which can be rendered if the user is not logged in; this just adds a safe navigation operator, so the `selectGroups` will just get returned as `undefined` instead of throwing a "user is nil" type exception